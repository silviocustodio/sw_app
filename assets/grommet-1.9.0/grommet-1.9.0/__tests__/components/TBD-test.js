// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import TBD from '../../src/js/components/TBD';

describe('TBD', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <TBD />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
