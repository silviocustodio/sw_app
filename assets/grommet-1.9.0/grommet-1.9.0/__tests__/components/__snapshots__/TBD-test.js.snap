// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TBD has correct default options 1`] = `
<div
  className="grommetux-tbd"
>
  TBD
</div>
`;
